src/asm/lexer.c

```diff
@@ -2389,13 +2390,15 @@ void lexer_CaptureRept(struct CaptureBody *capture)
 	lexerState->disableMacroArgs = false;
 	lexerState->disableInterpolation = false;
 	lexerState->atLineStart = false;
+	return terminated;
```

Comment:
Isn't `terminated` equivalent to `c != EOF`? (If so, a replacement + comment mentioning it would be useful.)

Reply:
Yes it is.